tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md

```diff
@@ -262,7 +262,7 @@ npm install --save-dev sqlite3
 
 3. To initialize the bookstore database with the defined domain model and sample data, run:
 ```Shell/Bash
-npm run deploy
+cds deploy --to sqlite
```

Comment:
This file contains the same string once again, in step 7, bullet point 3. Please also replace it there.
In addition these two files also need to be adapted with that change:
- https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-custom-logic/cp-cap-java-custom-logic.md (once)
- https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-reusable-service/cp-cap-java-reusable-service.md (twice)
- https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-reusable-service/run-deploy.png (screenshot)